packages/cxl-lumo-styles/scss/global.scss

```diff
@@ -73,7 +73,10 @@ ul {
  * headroom.js
  */
 .headroom {
-  will-change: transform;
+  // Let the browser decide what to do when some properties change in header, because setting it to "transform" causes issue.
+  will-change: auto;
+  // WordPress admin bar tweak.
+  top: var(--wp-admin--admin-bar--height, 0);
```

Comment:
> Is this somehow related to the issue? Proper admin bar integration is a separate target.

Not related to scroll issue, but related as well.

Because when user is logged in, and scrolls up, admin bar hides the header, so adding it would fix the hidden header issue, so thought could fix it in scroll related issue.

You can see secondary header is almost hidden behind `admin-bar`
![Uploading chrome_n5bwdr3aIP.png…]()

Reply:
Separate PR. Just resist the temptation, it creates more extra processing work than it optimizes.